index.html

```diff
@@ -10,14 +10,16 @@
     <title>Portfolio</title>
 </head>
 <body>
-    <div class="app-wrap">
+  
+    <section id="welcome">
+        <div class="app-wrap">
         <header class="app-header">
             <a class="logo" href="#">My Logo</a>
             <nav class="nav-bar">
                 <img class="menu" src="Union.png" alt="menu">
             </nav>
         </header>
-        <div class="content">
+        <div class="content welcome_content">
             <h1 class="salute">I'm Alexandra. Glad to see you!</h1>
             <p class ="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>

```

Comment:
- [ ] Kindly remove the border from the outside of the text to match the Figma design. :+1: 
![remove_border](https://user-images.githubusercontent.com/70483835/227504027-7c6d41bc-cc78-4000-9ad9-2c8786af782e.png)